### PR TITLE
Issue 4 log workout

### PR DESCRIPTION
**Pull request by rachelbeacham**
Screen Shot 2020-12-30 at 4 51 25 PM
Screen Shot 2020-12-30 at 4 51 41 PM
Screen Shot 2020-12-30 at 4 51 52 PM

### CODE REVIEW

File: server/public/styles.css [hunk 1]
```diff
@@ -60,3 +75,18 @@ select {
 .white-space {
   white-space: pre-wrap;
 }
+
+.hidden {
+  display: none;
+}
```
Comment (thebearingedge) [inline server/public/styles.css:81]: Bootstrap has display helpers.

https://getbootstrap.com/docs/4.5/utilities/display/#notation

File: server/public/styles.css [hunk 2]
```diff
@@ -38,11 +39,25 @@ select {
...
+
 .top-button {
   width: 5rem;
   height: 2rem;
 }
 
+.bottom-button {
+  position: fixed;
+  bottom: 0;
```
Comment (thebearingedge) [inline server/public/styles.css:58]: Bootstrap has position helpers.

https://getbootstrap.com/docs/4.5/utilities/position/

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, really awesome work so far. There are some relatively minor changes in my notes. Your UI is looking a bit weird in a couple spots though.
1. The `Back` and `Finish` buttons are suddenly touching the left and right edges of the viewport.
2. 1. Your Add Exercise form is looking a bit narrow. You can probably afford to let it expand horizontally such that its borders are spaced from the viewport edges like your exercise list item text.

File: server/public/styles.css (hunk 1 above)
Comment (rachelbeacham) [reply server/public/styles.css:81]: Didn't see that one! I noticed the invisible class but noticed that this left space as if the element was there

**Comment by rachelbeacham** [comment]
Screen Shot 2020-12-31 at 1 10 11 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**